region/maridia/inner-green.json

```diff
@@ -1505,7 +1519,7 @@
                   "note": [
                     "The sand does not impede Samus in G-Mode.",
                     "It is possible to jump to the tall pillar with nothing. It requires a very precise jump as late as possible.",
-                    "Alternatively, starting on the last pixel facing left, turning around and very rapidly jumping may be easier."
+                    "Alternatively, facing left while on the right-most pixel, then turning around and very rapidly jumping may be easier."
```

Comment:
This is subpixel dependant.  But you can adjust to a good value, and the tells are there if you have sharp eyes.
Feels like a 3 frame jump window to me.  Could be 2.
The problem is "very rapidly jumping"  sounded like like mashing jump helped somehow.

I learned running before jumping doesn't matter at all so you can stand one pixel from the right edge and do a forward spinjump.  But that also only works when jumping from the high subpixels of the farthest pixel. One frame jump.

I like the forward jump more.

Reply:
I think this is what you were saying?

Reply:
Right.  There is one subpixel precise spot that works and you can approach it facing forwards or backwards.

I think there is a `from[3]` strat that would use the same note change.

Reply:
Good catch. Thanks!